index.php

```diff
@@ -63,10 +63,10 @@
                             </a>
                         </div>
                         <div collapse="isCollapsed">
-                            <ul class="nav navbar-nav navbar-right" ng-if="data.currentUser">
+                            <ul class="nav navbar-nav navbar-right" ng-if="!data.currentUser">

```

Comment:
Looks good. In general, we could think about creating some global function like `userLoggedIn()` that returns a boolean and call that in these situations, but I'll leave it up to you if you want to bother. This should work fine as well.